ethexe/cli/src/service.rs

```diff
@@ -78,8 +103,17 @@ impl Service {
         .await?;
 
         let router_query = RouterQuery::new(&config.ethereum_rpc, ethereum_router_address).await?;
+
         let genesis_block_hash = router_query.genesis_block_hash().await?;
-        log::info!("👶 Genesis block hash: {genesis_block_hash}");
+        log::info!("👶 Genesis block hash: {genesis_block_hash:?}");
+
+        let validators = router_query.validators().await?;
+        log::info!("👥 Validators set: {validators:?}");
+
+        let threshold_percentage = router_query.threshold_percentage().await?;
```

Comment:
you can just use `.signingThresholdPercentage()`

Reply:
Cannot understand correctly what do you mean ?

Reply:
Done, use validatorsThreshold